coops-ui/src/components/Dashboard/TodoList.vue

```diff
@@ -131,14 +144,32 @@
                   </v-menu>
                 </template>
 
-                <v-btn v-else-if="isPending(item)"
-                  class="btn-resume-payment"
-                  color="primary"
-                  :disabled="!item.enabled"
-                  @click.native.stop="doResumePayment(item)"
-                >
-                  <span>Resume Payment</span>
-                </v-btn>
+                 <template v-else-if="isPending(item)">
+                  <v-btn class="btn-resume-payment"
+                    color="primary"
+                    :disabled="!item.enabled"
+                    @click.native.stop="doResumePayment(item)"
+                    data-test-id="btn-resume-payment">
+                    <span>Resume Payment</span>
+                    <!-- Cancel Payment -->
+                  </v-btn>
+                   <v-menu offset-y left>
+                    <template v-slot:activator="{ on }">
+                      <v-btn color="primary"
+                        v-on="on" id="pending-item-menu-activator" :disabled="!item.enabled"
+                        class="actions__more-actions__btn px-0"
+                      @click.native.stop  data-test-id="btn-pending-filing-menu">
```

Comment:
We don't need 'data-test-id' because we can already select this button via its id.

Reply:
As per John, it does not matter if a c omponent has id or not. The scripts will be looking at data-test-id and it should be there even if id is there so that the test script can be consistent

Reply:
OK, we will discuss this and decide what's best for everyone.

Reply:
In addition to our discussion this morning, I chatted with Roland Stearns, the Senior Tester we have for a bit. He said to do no extra work while he reviews whether existing selectors are sufficient. In other words, just leave this here for now.